ext/auto_tagger/main.php

```diff
@@ -245,6 +300,38 @@ private function add_auto_tag(string $tag, string $additional_tags)
                 AutoTaggerInfo::KEY,
                 "Added auto-tag for {$tag} -> {".implode(" ", $additional_tags)."}"
             );
+        }
+		// Now we apply it to existing items
+		$this->apply_new_auto_tag($tag);
+    }
+
+    private function update_auto_tag(string $tag, string $additional_tags): bool
```

Comment:
I think that's was part of an old change I made a long time ago to make updates to existing tags concatenate instead of replace.  I don't see it called anywhere in my own branch either so I must have decided to do it another way.